src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private Random random = new Random(100);
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ball.setBall(colorSupplier.getRandomColor(), random.nextInt());
```

Comment:
The method `nextInt()` without arguments will generate the entire range of `int`, which may include negative numbers. If you want to limit the number of the ball to a certain positive range, you should use `nextInt(bound)` with an appropriate bound value.